samples/cpp/common/utils/include/samples/classification_results.h

```diff
@@ -256,30 +171,18 @@ class ClassificationResultT {
         /** Print the result iterating over each batch **/
         std::ios::fmtflags fmt(std::cout.flags());
         std::cout << std::endl << "Top " << _nTop << " results:" << std::endl << std::endl;
-        for (unsigned int image_id = 0; image_id < _batchSize; ++image_id) {
-            std::wstring out(_imageNames[image_id].begin(), _imageNames[image_id].end());
-            std::wcout << L"Image " << out;
-            std::wcout.flush();
-            std::wcout.clear();
-            std::wcout << std::endl << std::endl;
+        for (size_t image_id = 0; image_id < _batchSize; ++image_id) {
+            std::string out(_imageNames[image_id].begin(), _imageNames[image_id].end());
+            std::cout << "Image " << out;
```

Comment:
Maybe `slog`?

Reply:
better try this in separate PR, as it may affect VPU tests, which gives us yet another month of delay